backend/core/llm_chain.py

```
@@ -3,490 +3,167 @@
-            model=CLAUDE_MODEL,
-            temperature=0.7,
-            timeout=REQUEST_TIMEOUT
-        )
-        logger.info(f"Claude model {CLAUDE_MODEL} initialized successfully (PRIMARY)")
-    except Exception as e:
-        logger.warning(f"Failed to initialize Claude (primary): {e}")
-        llms['claude'] = None
-
-    # Initialize Gemini as fallback
+        llms['claude'] = ChatAnthropic(model=CLAUDE_MODEL, temperature=0.7, timeout=REQUEST_TIMEOUT)
+    except Exception: llms['claude'] = None
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)